components/storage/blockdevice/COMPONENT_QSPIF/QSPIFBlockDevice.cpp

```diff
@@ -1045,7 +1047,7 @@ int QSPIFBlockDevice::_sfdp_detect_best_bus_read_mode(uint8_t *basic_param_table
             break;
         }
 
-        if (examined_byte & 0x80) {
+        if (examined_byte & 0x40) {
```

Comment:
It bothers me that these are magic numbers instead of definitions somewhere, but I'm not going to block the PR on them.

It would be nice if they could be un-magic-fied at some point.

Reply:
@cmonr - support for each mode is defiend as a certain bit of a certain byte. I did use a macro for the Byte location. I'll use a definition for the bits as well at later delivery.
![image](https://user-images.githubusercontent.com/36957743/54741446-2aca8000-4bc7-11e9-9219-36e93a0d0cb0.png)